prow/plugins/approve/approve_test.go

```diff
@@ -1089,6 +1085,39 @@ Approvers can cancel approval by writing ` + "`/approve cancel`" + ` in a commen
 </details>
 <!-- META={"approvers":[]} -->`,
 		},
+		{
+			name:                "don't suggest to /assign already assigned cjwagner",
+			hasLabel:            false,
+			files:               []string{"a/a.go", "c/c.go"},
+			comments:            []github.IssueComment{},
+			reviews:             []github.Review{newTestReview("Alice", "/approve", github.ReviewStateChangesRequested)},
+			selfApprove:         false,
+			needsIssue:          false,
+			lgtmActsAsApprove:   false,
+			reviewActsAsApprove: true,
+			githubLinkURL:       &url.URL{Scheme: "https", Host: "github.com"},
+
+			expectDelete:  false,
+			expectToggle:  false,
+			expectComment: true,
+			expectedComment: `[APPROVALNOTIFIER] This PR is **NOT APPROVED**
+
+This pull-request has been approved by: *<a href="" title="Approved">Alice</a>*
+To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **cjwagner** after the PR has been reviewed.
```

Comment:
it took me a while to figure out the logic of this unit test:
- cjwagner is the author of all test cases in this test
- this test case has `selfApprove: false`, so cjwagner didn't approve their own test
- then the PR suggests ask for approval from cjwagner

Logically speaking this is all correct, and this is perhaps the best we can do for this scenario in prod. Might not be in this PR, but maybe we should make the PR author configurable to cover more scenarios?